### PR TITLE
Date subscriptions

### PR DESCRIPTION
**Pull request by sienna-kopf**
# Crate Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Added date to subscriptions table and added subscriptionDateUpdate mutation for updating date. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`
# Message for reviewer:
`Your message here`
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: code/api/src/migrations/11-subscriptiondate.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```
Comment (rwilliams659) [inline code/api/src/migrations/11-subscriptiondate.js:4]: Great, we'll need to access this data - we might need your help figuring out exactly how we do that

File: code/api/src/modules/subscription/mutations.js [hunk 2]
```diff
@@ -17,6 +17,22 @@ export const subscriptionCreate = {
...
 
+// Subscription date update
+export const subscriptionDateUpdate = {
+  type: SubscriptionType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+    date: {
```
Comment (JoshSevy) [inline code/api/src/modules/subscription/mutations.js:28]: Awesome , great work we should be able to get the date change hooked up easily.

**Review by JoshSevy — APPROVED** [review]
Great work, Thanks team your the best!

File: code/api/src/seeders/4-subscriptions.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+'use strict';
```
Comment (rwilliams659) [inline code/api/src/seeders/4-subscriptions.js:1]: Great to have a seed file!

File: code/api/src/modules/subscription/mutations.js [hunk 4]
```diff
@@ -17,6 +17,22 @@ export const subscriptionCreate = {
   resolve: create
 }
 
+// Subscription date update
+export const subscriptionDateUpdate = {
```
Comment (rwilliams659) [inline code/api/src/modules/subscription/mutations.js:21]: We'll work to get this hooked up to the FE! Thanks for your hard work!

**Review by rwilliams659 — APPROVED** [review]
Thanks for the great work, BEEs!!!

**Review by Abdeboskey — APPROVED** [review]
This is really good! Thank you for making this happen, this should make it really simple for us to hook this up. Y'all are the best. 🐝

**Pull request merged**